pkg/runtime/conversion_generation_test.go

```diff
@@ -143,7 +145,7 @@ func compareBuffers(t *testing.T, generatedFile string, existing, generated byte
 }
 
 func TestNoManualChangesToGenerateConversions(t *testing.T) {
-	versions := []string{"v1beta3"}
+	versions := []string{"v1beta3", "v1"}
```

Comment:
I'm not sure about it.
Basically, since this test is pretty fast (less than 0.1s) I think we should run it for all explicitly defined APIs. That's why I'm putting those explicitly.